net_utils.js

```diff
@@ -187,10 +180,59 @@ exports.is_ip_in_str = function(ip, str) {
     return false;
 };
 
-exports.is_rfc1918 = function (ip) {
-    return (net.isIPv4(ip) && re_private_ipv4.test(ip));
+exports.is_private_ipv4 = function (ip) {
+
+    // RFC 1918, reserved as "private" IP space
+    if (/^10\./.test(ip)) return true;          // 10/8
```

Comment:
How did you benchmark?

I benchmarked by calling the is_private_ip function with 5 different IPs (2 hits, 3 misses), which in turn called the is_local_ipv4 and is_private_ipv4 functions (as existing in this PR). My understanding is that node compiles functions (including the regexes) in them, which would explain why I saw no performance difference.